### PR TITLE
Support `get_raw_config_parameter_value`

### PR DESCRIPTION
**Pull request by MindFreeze**
We need GET/SET for certification and only set was supported in the server

### CODE REVIEW

**Comment by MindFreeze** [comment]
@AlCalzone Pls review

File: src/lib/node/incoming_message.ts [hunk 1]
```diff
@@ -94,6 +94,14 @@ export interface IncomingCommandNodeSetRawConfigParameterValue
   valueFormat?: ConfigValueFormat;
 }
 
+export interface IncomingCommandNodeGetRawConfigParameterValue
+  extends IncomingCommandNodeBase {
+  command: NodeCommand.getRawConfigParameterValue;
+  parameter: number;
+  bitMask?: number;
+  allowUnexpectedResponse?: boolean;
```
Comment (AlCalzone) [inline src/lib/node/incoming_message.ts:102]: `allowUnexpectedResponse` is only really needed during the interview, which the driver does internally. I'd probably not expose it to applications.

**Review by AlCalzone — CHANGES_REQUESTED** [review]

**Review by AlCalzone — APPROVED** [review]

File: src/lib/node/command.ts [hunk 2]
```diff
@@ -8,6 +8,7 @@ export enum NodeCommand {
   abortFirmwareUpdate = "node.abort_firmware_update",
   pollValue = "node.poll_value",
   setRawConfigParameterValue = "node.set_raw_config_parameter_value",
+  getRawConfigParameterValue = "node.get_raw_config_parameter_value",
```
[the remark this answers is not in the document]
Comment (AlCalzone) [reply src/lib/node/command.ts:11]: Oh damn, should we have bumped the schema version aswell?

[the remark this answers is not in the document]
Comment (MartinHjelmare) [reply src/lib/node/command.ts:11]: It was already bumped in another PR in the release, so that's fine. We can just adjust the docs without a new release.

**Pull request merged**